### PR TITLE
Flatten machine and service commands

### PR DESCRIPTION
**Pull request by AlexisBruemmer**
New Command Format:

machines - list machines in a model
list-machines - list machines in a model
add-machine - adds machine to a model
remove-machine - removes a machine from a model
show-machine - shows details on a machine

get-config - gets the config values for a service
set-config - sets the config value for a service
get-constraints - gets the constraints for a service
set-constraints - sets the constraints for a service
add-unit - add unit[s] to a service
remove-unit - remove unit from a service

### CODE REVIEW

File: cmd/juju/commands/main_test.go [hunk 1]
```diff
@@ -256,9 +254,11 @@ var commandNames = []string{
 	"list-users",
 	"login",
 	"machine",
+	"machines",
 	"publish",
 	"remove-all-blocks",
-	"remove-machine",  // alias for destroy-machine
+	"remove-machine", // alias for destroy-machine
```
Comment (wallyworld) [inline cmd/juju/commands/main_test.go]: the alias comment is not needed now

File: cmd/juju/commands/main_test.go [hunk 2]
```diff
@@ -236,6 +231,7 @@ var commandNames = []string{
 	"expose",
 	"generate-config", // alias for init
 	"get-config",
+	"get-configs",
```
Comment (wallyworld) [inline cmd/juju/commands/main_test.go]: You may disagree, but i still thing config is a collective noun like fish or sheep.
You wouldn't say "Look at those sheeps". Similarly, get-configs seems quite wrong.

File: cmd/juju/machine/show.go [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+	isoTime   bool
+	machineId []string
+	api       statusAPI
+}
+
+// Info implements Command.Info.
+func (c *showMachineCommand) Info() *cmd.Info {
+	return &cmd.Info{
+		Name:    "show-machine",
+		Args:    "<machineID[s]> ...",
```
Comment (wallyworld) [inline cmd/juju/machine/show.go]: "<machineID> ..."

(see cmd/juju/model/unset.go for an example). We're best to be consistent. Also update the showMachineCommandDoc above

File: cmd/juju/service/set.go [hunk 4]
```diff
@@ -71,6 +76,13 @@ func (c *setCommand) Init(args []string) error {
 		return errors.New("cannot specify --config when using key=value arguments")
 	}
 	c.ServiceName = args[0]
+	if c.SetDefault == true {
```
Comment (wallyworld) [inline cmd/juju/service/set.go]: delete the == true

**Comment by wallyworld** [comment]
LGTM with a few little fixes.

File: cmd/juju/commands/main_test.go (hunk 2 above)
Comment (AlexisBruemmer) [inline cmd/juju/commands/main_test.go]: Our current direction is "all commands have singular and plural forms"; I would like to leave this until we have consensus next week.  It is easy enough to update later.

**Comment by AlexisBruemmer** [comment]
$$merge$$

**Pull request merged**